server/middleware.go

```diff
@@ -42,6 +44,32 @@ func JSONToHTTP(ep JSONEndpoint) http.Handler {
 	})
 }
 
+// ContextToHTTP is a middleware func to convert a ContextHandler an http.Handler
+func ContextToHTTP(ctx context.Context, ep ContextHandler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.Body != nil {
+			defer func() {
+				if err := r.Body.Close(); err != nil {
+					Log.Warn("unable to close request body: ", err)
+				}
+			}()
+		}
+
+		var (
+			cancel context.CancelFunc
```

Comment:
Mind changing it to a one liner since we only have a single variable being declared?